src/data/stream/basic.lean

```diff
@@ -1,23 +1,523 @@
 /-
-Copyright (c) 2020 Gabriel Ebner, Simon Hudon. All rights reserved.
+Copyright (c) 2015 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Gabriel Ebner, Simon Hudon
+Authors: Leonardo de Moura
```

Comment:
Shouldn't we merge these author lists?

Reply:
@gebner  and @cipher1024 wrote <30 LOC. Of this code, the following lemmas survived: `stream.inhabited`, docstring of `stream.take` (in `defs`), `stream.length_take`, `stream.corec_state` (in `defs`). As far as I understand, usually we do not add people who contributed a few lemmas to the authors list.